### ISSUE TITLE
App approval in itunnes connect

### ISSUE THREAD

**Issue by fobetor** [bakerframework/bakercloud-ce-console#3]
Helo again... I have looked in the magrocket issues, but I dont have the following issue clear... Let me describe it.
 I have an approved version of baker in the app store. This version (1.0) used free issues and does not use magrocket.... Not released jet since i wanted to use magrocket, so I prepared magrocket and baker and resubmited the app. 
 I prepared an in-app purchase issue in itunes connect (description ,price, date etc..), but i didnt add it in magrocket or added the issue (for instance its not done, it's a future plan for non free)
 So... Apple rejected the binary because:
"We have begun the review of your app but aren't able to continue because we can't locate the Non-Consumable In App Purchase within your app. "
 THE Question is:
 If I prepare the issue description in itunnes coonect, but I dont add any real issue and i dont add it in magrocket, the reviwer at apple would not be able to see it in my app, and he will reject it...?? Because it is not present in my app shelf?
 Since apple review takes about a week to approve a issue/ in-app purchase... If i prepare magrocket to display the in app purchase so reviewer can see the issue/price etc.. In the shelf, if the ap is Live, any user would see that magazine... But it cant be purchased because it is not approved by apple..... A mess?
 I'm missing something about the correct process to make this have sense for apple?
Thank you

**Comment by nin9creative** [comment]
I've been working with a publisher and they typically add the Issues to MagRocket in pending status while they work on them etc...

They submit issues weekly to be approved prior to actually finishing the development of the issue, so I don't know what Apple is expecting
to see in this case.  If you are prepping an Issue for release on your app, then they will/can only see the information provided for the Non Consumable
In App Purchase that you are submitting.

What shows up in your shelf now?  The Free Issues?  What did you submit to Apple for Review?  Did you say that it was just a fix/bug release or update?

If no issues show up for them to review I can see the problem, however if something shows up then you might be able to reply saying that you are still
developing the Paid In App purchase but you need submit an app update first to prepare for the paid content.

Andrew

**Comment by fobetor** [comment]
Andrew
 Now, they can see a free issue. As when I had the app apporved for first time.
 The last app version I sent, I made an in-app purchase in previewing that I will put one non-free issue in the future, but it was not in the baker shelf.... and they answered : 
"We have begun the review of your app but aren't able to continue because we can't locate the Non-Consumable In App Purchase within your app. "
That's why I asked you abput this. Because I dont understand the process they are using. It seems as they are especting to see an in-app buying issue with the price etc in the shelf.
 I have deleted the non-consumable issue in the in app purchase, and left only the free one now... I'll tell you what happens next... ;-)
 The idea I got, was to use the process you mentioned above...
